fhttp/httprunner_test.go

```diff
@@ -471,6 +471,38 @@ func TestAbortOn(t *testing.T) {
 	}
 }
 
+func TestMaxConnectionReuse(t *testing.T) {
```

Comment:
see the other tests that have an array of input and expected output so you can test as mentioned above something like

n=10 (instead of relying on duration)

and ranges from [1,1] to [10,10]

Reply:
updated